.github/workflows/master.yml

```diff
@@ -297,6 +297,7 @@ jobs:
       - license_check
       - go_test_shim
       - test_oidc
+      - unit_test
```

Comment:
These are not unit tests AFAIK. These are the provider integration tests that are not examples.